lib/src/rules/sort_child_properties_last.dart

```diff
@@ -52,8 +52,13 @@ return Scaffold(
     title: Text(widget.title),
   ),
   body: Center(
-    mainAxisAlignment: MainAxisAlignment.center,
+    floatingActionButton: FloatingActionButton(
```

Comment:
Thanks for doing this.

Unfortunately, both examples are wrong :-/. floatingActionButton is not a property of `Center` widget. It is a property of `Scaffold`.

Here's the proper bad example:

```dart
return Scaffold(
  appBar: AppBar(
    title: Text(widget.title),
  ),
  body: Center(
    child: Column(
      children: <Widget>[
        Text(
          'You have pushed the button this many times:',
         ),
        Text(
          '$_counter',
          style: Theme.of(context).textTheme.display1,
         ),
      ],
      mainAxisAlignment: MainAxisAlignment.center,
    ),
    widthFactor: 0.5,
  ),
  floatingActionButton: FloatingActionButton(
    child: Icon(Icons.add),
    onPressed: _incrementCounter,
    tooltip: 'Increment',
  ),
);
```

here's the good example:

```dart
return Scaffold(
  appBar: AppBar(
    title: Text(widget.title),
  ),
  body: Center(
    widthFactor: 0.5,
    child: Column(
      mainAxisAlignment: MainAxisAlignment.center,
      children: <Widget>[
        Text(
          'You have pushed the button this many times:',
         ),
        Text(
          '$_counter',
          style: Theme.of(context).textTheme.display1,
         ),
      ],
    ),
  ),
  floatingActionButton: FloatingActionButton(
    child: Icon(Icons.add),
    onPressed: _incrementCounter,
    tooltip: 'Increment',
  ),
);
```

Reply:
Gotcha, even worse then. I updated to a correct set of examples.